sssd_test_framework/roles/ipa.py

```diff
@@ -455,19 +455,28 @@ def add(
     def modify(
         self,
         *,
+        first: str | None = None,
+        last: str | None = None,
         uid: int | None = None,
         gid: int | None = None,
         password: str | None = None,
         home: str | None = None,
         gecos: str | None = None,
         shell: str | None = None,
         user_auth_type: str | list[str] | None = None,
+        idp: str | None = None,
+        idp_user_id: str | None = None,
+        password_expiration: str | None = None,
     ) -> IPAUser:
         """
         Modify existing IPA user.
 
         Parameters that are not set are ignored.
 
+        :param first: First name of user
```

Comment:
.

Seems like if there is not no comma, a period at the end, with a comma, period is dropped. 

:param first:  First name.
:param first: First name, defaults to None
:type or rtype: no periods

Reply:
updating